sdk/src/main/java/io/dapr/client/DaprClientBuilder.java

```diff
@@ -172,6 +175,8 @@ private DaprClient buildDaprClientGrpc() {
    * Creates and instance of DaprClient over HTTP.
    *
    * @return DaprClient over HTTP.
+   *
+   * @deprecated HTTP support is being deprecated; Use the gRPC builder method (buildDaprClientGrpc) instead. This will be removed in a future release.
    */
   private DaprClient buildDaprClientHttp() {
     return new DaprClientHttp(this.daprHttpBuilder.build(), this.objectSerializer, this.stateSerializer);
```

Comment:
Log a warning that http client is deprecated and will be removed in future release...

Reply:
I had thought about that but we don't have a standard logging mechanism in the client (I looked and it appears we don't log anything except in the tests). I would hesitate to take additional dependencies unless we are going to add logging elsewhere in the client (which might not be a bad idea but we don't currently). 

Thoughts?

Reply:
That is a great point. We don't have any logging in the SDK on purpose right now to avoid bringing dependencies that might conflict with apps' dependencies. I see 2 options here:

1. Depend on SLF4J, so users can still pick their own logging solution as SLF4J is an abstraction. Of course, the version of SLF4J can conflict and users will need to pick one (probably the latest).
2. Use System.out.Println() - I am OK since most users run this on containers or local mode where logs go to STDOUT or STDERR.

Reply:
I'm fine with just printing to STDOUT for the moment -- while most of the universe uses SLF4J, and it's pretty stable, there are enough issues with dependencies in general that it would be great if we could avoid piling on any more.

Reply:
Added a `System.out.println` statement when constructing `DaprClientHttp` objects.

Reply:
Added a `System.out.println` statement when constructing a `DaprClientHttp` object